pkg/securitypolicy/securitypolicy_internal.go

```diff
@@ -8,11 +8,12 @@ import (
 
 // Internal version of SecurityPolicy
 type securityPolicyInternal struct {
-	Containers            []*securityPolicyContainer
-	ExternalProcesses     []*externalProcess
-	AllowPropertiesAccess bool
-	AllowDumpStacks       bool
-	AllowRuntimeLogging   bool
+	Containers              []*securityPolicyContainer
```

Comment:
Hi Maksim, the continued addition of bools to this function signature makes a good case for refactoring. I recommend a type or similar construct to avoid unreadable code at the function invocation: 

code, err = marshalRego(
			securityPolicy.AllowAll,
			containers,
			[]ExternalProcessConfig{},
			true,
			true,
			true,
			true,

Reply:
yeah, we have "post feature complete" list of improvements and this one is on it 😄